components/omnibox/browser/topsites_provider_unittest.cc

```diff
@@ -19,7 +20,7 @@ class TopSitesProviderTest : public testing::Test {
                             metrics::OmniboxEventProto::OTHER,
                             classifier_);
     return input;
- }
+  }
```

Comment:
Nit: whitespace here seems unintentional.

Reply:
It's actually an intentional lint correction, only changed files are tested in a PR so this file was ignored before.  👍